### PR TITLE
allow numbers in error type name

### PR DESCRIPTION
**Pull request by pnepywoda**

### CODE REVIEW

File: errors/src/main/java/com/palantir/remoting/api/errors/ErrorType.java [hunk 1]
```diff
@@ -30,7 +30,7 @@
 @ImmutablesStyle
 public abstract class ErrorType {
 
-    private static final String UPPER_CAMEL_CASE = "(([A-Z][a-z]+)+)";
+    private static final String UPPER_CAMEL_CASE = "(([A-Z][a-z][0-9]+)+)";
```
Comment (uschi2000) [inline errors/src/main/java/com/palantir/remoting/api/errors/ErrorType.java]: Doesn't access name `Foo`

Comment (uschi2000) [reply errors/src/main/java/com/palantir/remoting/api/errors/ErrorType.java]: (also indicates that the tests are quite good enough)

Comment (uschi2000) [reply errors/src/main/java/com/palantir/remoting/api/errors/ErrorType.java]: (also indicates that the tests are quite good enough)

**Comment by pnepywoda** [comment]
lolz thanks tests

File: errors/src/main/java/com/palantir/remoting/api/errors/ErrorType.java (hunk 1 above)
Comment (uschi2000) [reply errors/src/main/java/com/palantir/remoting/api/errors/ErrorType.java]: OK names: Foo, Foo123, FooBar, Foo3Bar29
Bad names: 123, 123Foo

Comment (pnepywoda) [reply errors/src/main/java/com/palantir/remoting/api/errors/ErrorType.java]: agreed. thats what this does

**Comment by uschi2000** [comment]
can you add a simple test please?

**Comment by pnepywoda** [comment]
https://github.com/palantir/http-remoting-api/pull/52

**Pull request closed without merging**